src/wp-content/themes/twentyeleven/footer.php

```diff
@@ -25,7 +25,14 @@
 			?>
 
 			<div id="site-generator">
-				<?php do_action( 'twentyeleven_credits' ); ?>
+				<?php
+				/**
+				 * Fires before the Twenty Eleven credits in the footer.
```

Comment:
```suggestion
				 * Fires before the Twenty Eleven footer text for footer customization.
```

I still think each of these four themes should say "footer text" instead of "credits" because the privacy policy link is between the action and the WordPress credit. I modeled this after the description in Twenty Fifteen.